shared/constants/types/more.js

```diff
@@ -85,3 +85,9 @@ export type DumbComponentMap<C: Component<*, *, *>> = {
     [key: string]: PropsOf<C>,
   },
 }
+
+export type LooseRecord<T> = T & {
```

Comment:
I don't like it. Maps are looser than records (in our existing terminology). And it makes it sound like this is a combination of Map and Record. It's really just a Record (but a more accurate type)